src/Chainweb/Version/Utils.hs

```diff
@@ -410,21 +410,26 @@ expectedBlockCountAfterSeconds
     -> cid
     -> Seconds
     -> Double
-expectedBlockCountAfterSeconds v cid s = 1 + (int s / int r) - int gh
+expectedBlockCountAfterSeconds v cid s = max 0 (1 + (int s / int r) - int gh)
```

Comment:
This can happen at block heights that are smaller than the genesis height of new chains. For instance, for instance after 1 second `int s / int r` is expected to be very small. If a chain has a genesis height of, say, 10000, the value of `(int s / int r) / int gh` would become negative and the expected block height should be set to 0, which is what the `max 0` ensures.

Reply:
👍